plugins/modules/website.ps1

```diff
@@ -1,27 +1,280 @@
 #!powershell
 
-# Copyright: (c) 2024, Ansible Project
+# Copyright: (c) 2025, Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 #AnsibleRequires -CSharpUtil Ansible.Basic
 
+# Define Bindings Options
+$binding_options = @{
+    type = 'list'
+    elements = 'dict'
+    options = @{
+        ip = @{ type = 'str' }
+        port = @{ type = 'int' }
+        hostname = @{ type = 'str' }
+        protocol = @{ type = 'str' ; default = 'http' ; choices = @('http', 'https') }
+        ssl_flags = @{ type = 'str' ; default = '0' ; choices = @('0', '1', '2', '3') }
```

Comment:
Are these flags representing some more human friendly value. It's hard to tell what each number means without having to look it up.